module/retrieval/mock.go

```diff
@@ -1,9 +1,8 @@
-package mock
+package retrieval
```

Comment:
oh interesting, i think this might of been causing me some issues last night - was going the other route with moving this into mock itself but ran into the retrieval dependency anyways with `ResultContent`, this makes mroe sense